src/compiler/crystal/codegen/link.cr

```diff
@@ -18,6 +18,12 @@ module Crystal
       has_pkg_config = nil
 
       String.build do |flags|
+        # Add the default path as -L flags: important on FreeBSD,
+        # where the default cc doesn't use /usr/local/lib by default
+        library_path.each do |path|
+          flags << " -L#{path}"
+        end
```

Comment:
Maybe? But I don't think adding `-L/usr/lib -L/usr/local/lib` can break anything anywhere… Do people *not* add this to all linker invocations?